tests/test_device.py

```diff
@@ -99,20 +116,30 @@ def mock_device(monkeypatch):
         yield Device()
 
 
-
-
 class TestDeviceSupportedLogic:
     """Test the logic associated with the supported operations and observables"""
 
+    # pylint: disable=no-self-use, redefined-outer-name
```

Comment:
You can move these to the top of the module, avoids needing to repeat them for each class

Reply:
Thanks!